MessageViewControllerTests/MessageTextViewTests.swift

```diff
@@ -122,7 +122,7 @@ class MessageTextViewTests: XCTestCase {
         let font = UIFont.systemFont(ofSize: 60)
         view.font = font
         XCTAssertEqual(view.defaultFont, font)
-        XCTAssertEqual(view.defaultTextAttributes[NSAttributedStringKey.font.rawValue] as! UIFont, font)
+        XCTAssertEqual(view.defaultTextAttributes[NSAttributedString.Key.font] as! UIFont, font)
```

Comment:
Nit: you can omit the `NSAttributedString.Key` prefixes here; they are inferred.